website/.eslintrc.json

```
@@ -0,0 +1,4 @@
+{
+  "root": true,
+  "extends": "next/core-web-vitals"
```

Hmm, let me try removing it for now.
```suggestion
  "root": true
```

---

@roguesherlock
This is temporary workaround, let's look for a better solution. We should run eslint on `website`.

---

Ah, it wasn't okay. I fixed in #343.